cudapoa/src/cudapoa_batch.cuh

```diff
@@ -625,6 +630,7 @@ protected:
 
     // Use banded POA alignment
     bool banded_alignment_;
+    bool adaptive_banding_;
```

Comment:
[nit] Should we rename all the `[cuda_]banded_alignment` and `[cudap]adaptivie_banding/ed` to one convention? I can push that change in if needed.

Reply:
I agree, will be good to have them consistent. Please make the changes if you have a chance.